text/0034-telemetry-policies.md

```diff
@@ -191,6 +191,8 @@ The collected data is sent to [Segment](https://segment.com/). Segment is a plat
 | `clear_all` | `true` if `DELETE /indexes/:indexUid/documents` endpoint was used in this batch, otherwise `false` | false | `Documents Deleted` |
 | vector_store | Whether the [vector store](./0193-experimental-features.md#vector-store) feature is enabled. | `true` | `Experimental features Updated` |
 | score_details | Whether the [score details](./0193-experimental-features.md#score-details) feature is enabled. | `true` | `Experimental features Updated` |
+| scoring.show_ranking_score | Was `showRankingScore` used in the aggregated event? If yes, `true`, otherwise `false` | `false` | `Documents Searched POST`, `Documents Searched GET` |
```

Comment:
Should it also be branched on the Multi-Search Event?

Reply:
Not sure what you mean: it is triggered by the Multi Search Event, but we don't write this in this table for other similar events, so I didn't as well.

<details>

`show_ranking_score` is an option of an individual search query inside of a multi search request. Any telemetry attached to the `Documents Searched POST` will also be sent in the multi search route, by construction.

For the table, I followed the existing convention that doesn't mention multi search for events attached to the search query (e.g. `formatting.highlight_post_tag` also doesn't mention multi search event). We could change this convention, but it is not directly related to this PR (and IMO it would add a lot of noise to the table).
</details>

Reply:
If a telemetry point is associated with an event, it should be mentioned in the list of related events; this is the table the documentation team is providing on the doc for the public. If some are missing this is a miss from our side 🥹

Reply:
so I checked the implementation and **I was mistaken**

It doesn't look like we are sending analytics about the individual queries when using the multi-search route.

So the table as written is the spec is correct (as in, the implementation does the same as in the spec).

Not sure if this is a good thing or if we should send some analytics about the searches in the multi-search route but at least now the spec and implementation agree.

Reply:
@dureuill would that mean that if (hypothetically) users only use scores to generate a unified list when running multi-indexed searches we wouldn't get any data on Amplitude? That sounds like we'd be missing on tracking one of the top use cases 🤔

Reply:
> @dureuill would that mean that if (hypothetically) users only use scores to generate a unified list when running multi-indexed searches we wouldn't get any data on Amplitude? That sounds like we'd be missing on tracking one of the top use cases

Yes that's what would happen in the current implementation.

Reply:
To move this forward, we could either:

1. add all the telemetry of the single search route for each search query used in the multi-search route
  - pros: easy to do, won't require additional work in the future
  - cons: hard to know if a query was done in multi or single search context, adds a lot of telemetry
2. The above, but add a bit of context to be able to differentiate between queries in multi or single search contexts.
  - pros: won't require additional work in the future, provide contexts
  - cons: hard to do, adds a lot of telemetry
3. Hard-code counters of the number of requests using `showRankingScore` (resp. `showRankingScoreDetails`) in the multi-search route
   - pros: easy to do, provides context, only adds what we need
   - cons: duplicates ways of sending analytics, will require additional work in the future if we want to add other analytics from single search to multi search

What would your preference be?

Reply:
it now means that the analytics related to one feature are now sent by two distinct codepaths. It has a higher maintenance cost and could potentially lead to bugs where one instance of the analytics is updated and not the other (think, renaming the analytics flag for instance).

Reply:
Opened https://github.com/meilisearch/meilisearch/pull/3906 to fix this with (3)

Reply:
```suggestion
| scoring.show_ranking_score | Was `showRankingScore` used in the aggregated event? If yes, `true`, otherwise `false` | `false` | `Documents Searched POST`, `Documents Searched GET`, `Documents Searched by Multi-Search POST` |
```

to apply after https://github.com/meilisearch/meilisearch/pull/3906 lands

Reply:
Awesome thanks!